recipes/py2dm/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "py2dm" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py2dm-{{ version }}.tar.gz
+  sha256: 447e3af1e5adfc7a898acd5e5744ab58f86d995cf615147629fec7b4bf605be7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+
+requirements:
+  build:
+    - typing-extensions   # [py<38]
+    - cached-property     # [py<38]
+    - setuptools
+  host:
+    - python
+    - typing-extensions   # [py<38]
+    - cached-property     # [py<38]
```

Comment:
These are likely not needed in host:
```suggestion
```

Reply:
Turns out they are needed, otherwise there's a build error while running setup.py egg_info. I don't know enough about python module packaging to understand why

Reply:
Indeed, [upstream is importing itself during the build](https://github.com/leonhard-s/Py2DM/blob/b67fac13a3465323a218f52f62885ab38bb727c1/setup.py#L6) so this is fine as-is.

For reference: Ideally the `version = attr: py2dm.__version__` syntax would be used instead ([see here](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html)) as that doesn't require the package to be importable.